lib/restforce/db/synchronizer.rb

```diff
@@ -49,10 +49,11 @@ def run(changes)
       #
       # Returns nothing.
       def update(instance, accumulator)
+        return unless accumulator.changed?(instance.attributes)
```

Comment:
In seeing this in use, I wonder if the method should be `differs_from?` or somesuch, as it reads a little more smoothly.